Includes/Rosetta/Tasks/SimpleTasks/EnqueueNumberTask.hpp

```diff
@@ -0,0 +1,42 @@
+// This code is based on Sabberstone project.
+// Copyright (c) 2017-2019 SabberStone Team, darkfriend77 & rnilva
+// RosettaStone is hearthstone simulator using C++ with reinforcement learning.
+// Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
+
+#ifndef ROSETTASTONE_ENQUEUE_NUMBER_TASK_HPP
+#define ROSETTASTONE_ENQUEUE_NUMBER_TASK_HPP
+
+#include <Rosetta/Tasks/ITask.hpp>
+
+namespace RosettaStone::SimpleTasks
+{
+//!
+//! \brief EnqueueNumberTask class.
+//!
+//! This class represents the task for repeating some tasks by
+//! a certain number of times.
+//!
+class EnqueueNumberTask : public ITask
+{
+ public:
+    //! Constructs task with given \p tasks and \p isSpellDamage.
+    //! \param tasks A task vector to repeat.
+    //! \param isSpellDamage true if it is spell damage, and false otherwise.
+    EnqueueNumberTask(std::vector<ITask*> tasks, bool isSpellDamage = false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'EnqueueNumberTask' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4719547)